tests/resources/hdfs-kerberos/datanode-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hdfs-datanode
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: hdfs-datanode
+  template:
+    metadata:
+      annotations:
+        pod.beta.kubernetes.io/hostname: datanode
+      labels:
+        name: hdfs-datanode
+        kerberosService: datanode
+    spec:
+      containers:
+      - command:
+        - start-datanode.sh
+        name: datanode
+        image: lembiewski/hdfs-kerberos:v1.0.0
```

Comment:
I think it makes sense to provide source Dockerfiles so that we can easily rebuild the images in case the repo changes. Although we should find a proper repo for the images.

Reply:
done